Update sso.md (#7371)
segmentio/segment-docs: src/segment-app/iam/sso.md

For most customers, Segment recommends requiring SSO for all users. If you do not require SSO, users can still log in with a username and password. If some members cannot log in using SSO, Segment also supports SSO exceptions.

> These options are off by default, but configurable on the "Advanced Settings" page. Login via SSO to toggle "Require SSO" setting. 

Reviewer: ```suggestion
These options are off by default, but you can configure them on the **Advanced Settings** page. Log in using SSO to toggle the **Require SSO** setting. 
```
